airflow/jobs/__init__.py

```diff
@@ -16,3 +16,7 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+import airflow.jobs.backfill_job  # noqa
+import airflow.jobs.base_job  # noqa
+import airflow.jobs.local_task_job  # noqa
+import airflow.jobs.scheduler_job  # noqa
```

Comment:
Instead of `noqa` we can add this file to per-file-ignores in .flake8 . WDYT?

Reply:
I prefer to keep ignore close to where the problem is. Nobody looks into the configuration file. Ignoring files in the configuration file is helpful if you have generated or similar files. Then you do not want to modify the file because ... the file is generated, so its modification is difficult. WDYT?

Reply:
What benefit do you see from moving this to a .flake8 file?

Reply:
> What benefit do you see from moving this to a .flake8 file?

Less code-obfuscating comments. We've already has files in this file. But no strong opinion on that.

Reply:
It's ok with noqa :)